inst/odin/carehomes.R

```diff
@@ -173,545 +174,595 @@ prob_admit_conf[] <- p_admit_conf * psi_admit_conf[i]
 #### flow out of S ####
 
 ## new infections
-n_S_progress[, ] <- rbinom(S[i, j], p_SE[i, j])
+
+## Compute the new infections with multiple strains using nested binomials
+n_S_progress[, 1, ] <- rbinom(S[i, k], p_SE[i, 1, k])
+n_S_progress[, 2:n_strains, ] <-
+ rbinom(S[i, k] - sum(n_S_progress[i, 1:(j - 1), k]), p_SE[i, j, k])
+
```

Comment:
From what I understand, `p_SE[i, j, k]` represents the probability of being infected with strain `j`, but these are not mutually exclusive? And to me here it looks like we are giving automatic priority to strains with lower j.

Suppose we have three strains such that 
`p_SE[i,1,k] = 0.25`
`p_SE[i,2,k] = 0.5`
`p_SE[i,3,k] = 0.75`

As I understand the formulation we have, strain 1 would infect about 25%, then strain 2 would infect about 50% of 75% = 37.5%, and strain 3 would infect about 75% of 37.5% = 28.1%. This despite strain 3 having a greater force of infection. Do we want this or do we want something that would give more proportional prioritisation?

Reply:
Would it be better to work out first the total number infected by all strains combined, and then determine how many were infected by each strain?

Reply:
Fair point. I think with a small enough time step we should never get to those high probabilities but you are right that this is not ideal. 
@richfitz  this actually means the example I gave in the multinomial odin issue is not good. 
@edknock  what is your suggestion to do this properly? draw the total number of infections and then split them proportionally between strains?

Reply:
That sounds like drawing samples without replacement which is the multivariate hypergeometric - I'd rather not implement that here yet as it's really awkward

Reply:
I think binomial followed by multinomial should work
I will try to implement tonight and get you to review

Reply:
OK great! I will see if I get time to make multinomial easier soon

Reply:
I tried to fix this - @edknock if you could check you're happy with this that would be great